restaurant-front/src/modules/shared/components/order-view/order-view.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { Order } from '../../models/order';
+import { OrderedItem } from '../../models/orderedItem';
+import { FinishDialogComponent } from '../finish-dialog/finish-dialog.component';
+
+
+@Component({
+  selector: 'app-order-view',
+  templateUrl: './order-view.component.html',
+  styleUrls: ['./order-view.component.scss']
+})
+export class OrderViewComponent implements OnInit {
+  
+  @Input() typeBtn: string="";
```

Comment:
Why is this an input?

Reply:
Because this component is for orders and they can be new or accepted, and depending on it button is different.